generators/app/templates/src/app/index.ejs

```diff
@@ -20,13 +13,6 @@ const App = () => {
   
   useEffect(() => SplashScreen.hide(), []);
   useEffect(() => {
-    <%_ if(features.pushnotifications) { _%>
-    configPushNotifications(dispatch);
-    alertIfHuaweiDevice();
-    <%_ } _%>
```

Comment:
Why you delete this config? I think It's ok to configure Push Notifications right here:
```
  <%_ if(features.pushnotifications) { _%>
    configPushNotifications();
  <%_ } _%>
```